roles/tower-setup/defaults/main.yaml

```diff
@@ -16,14 +16,14 @@
 #  You should have received a copy of the GNU General Public License
 #  along with ansible-tower-cicd.  If not, see <http://www.gnu.org/licenses/>.
 
+tower_admin_username: admin
+tower_admin_password: password
 tower_setup_basename: ansible-tower-setup
 tower_setup_version: 3.5.0-1
 tower_setup_archive: "{{ tower_setup_basename }}-{{ tower_setup_version }}.tar.gz"
 tower_setup_release_url: "https://releases.ansible.com/ansible-tower/setup/{{ tower_setup_archive }}"
 tower_setup_config:
   create_preload_data: false
-  admin_username: admin
-  admin_password: password
```

Comment:
admin_password needs to stay in tower_setup_config because there is a pre-flight check in the tower setup:
```
    - name: Preflight check - Passwords must be defined for a fresh install
      fail:
        msg: 'Please configure passwords in the inventory file before running setup'
      when: "(pg_password|default('') == '') or (admin_password|default('') == '')"
      delegate_to: localhost
      run_once: True
```

tower_setup_config gets written to /etc/tower-setup/extra-vars.yaml which is then included when running setup.sh.
``admin_password: "{{ tower_admin_password }}"`` is probably fine.

Reply:
Good catch. It looks like I mixed up `tower_setup_config` and `tower_setup_cli_config` when I wrote this.